### ISSUE TITLE
Show password button doesn't have enough constrast when active

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce#25615]
**To Reproduce**
1. Set your theme to 2020 or 2019.
2. Go to _My Account_ page (probably under `/my-account`).
3. Click on _Show password_ button.
4. Notice the button color when activated is `#e8e8e8` while the background is `#ffffff`. That is not enough contrast to pass accessibility tests: https://webaim.org/resources/contrastchecker/?fcolor=E8E8E8&bcolor=FFFFFF.

**Screenshots**
![imatge](https://user-images.githubusercontent.com/3616980/74071818-3dc78800-4a05-11ea-9831-1954d6675334.png)

**Expected behavior**
The gray color should be darker to ensure there is enough contrast.

Relevant line of code:
https://github.com/woocommerce/woocommerce/blob/master/assets/css/woocommerce-layout.scss#L430

**Comment by aryaanlambe** [comment]
Hi!
Can I have a go at this issue?

**Comment by juliaamosova** [comment]
>Can I have a go at this issue?

Hi @aryaanlambe, sure! You are welcome to work on any issue in this repository and submit a PR. One thing though - make sure to assign an issue to yourself to indicate that you are working on it so no one else would work on it as well.

**Comment by aryaanlambe** [comment]
Awesome! I'll get working on it soon :smile:

**Comment by juliaamosova** [comment]
Closed in https://github.com/woocommerce/woocommerce/pull/25625.
